style.css

```diff
@@ -177,7 +177,7 @@ nav .navbar {
   margin-bottom: 50px;
```

Comment:
- [ ] `required`
- [ ] ❗ Note that this applies to all the pop-up cards
- [ ] Amazing work so far! 👏 
- [ ] However Just a few adjustments
- [ ] Please make the Pop-up picture to fit the screen enough. so that the user doesnt have to scroll down to see the rest of the pop-up
- [ ] also please add the blur in the background
- [ ] And i Suggest you make the `X` closing button as close as the one in the figma template
- [ ] please see screenshot below

`Your template`
![image](https://user-images.githubusercontent.com/66230306/216691322-bfb5c580-673c-4a4d-ba82-1b5d9a1a37c5.png)

`figma template`
![image](https://user-images.githubusercontent.com/66230306/216691804-529ae018-0a51-4240-8068-af0afc464796.png)